### DISCUSSION TITLE
No "popup" windows

### DISCUSSION THREAD

**Discussion by dusty-phillips** [LazyVim/LazyVim#3175] [Q&A] [answered]
After updating my plugins this morning, I can't see any confirmation dialogs. Easiest repro is to create a new file and add some text to it, then type `:q`. Instead of popping up a yes/no widget, it just hangs. Also not seeing it when trying to confirm changes to mini.files.

I can interact with the window (e.g. y or no does what is expected when the quit dialog is not visible), but can't see it.

To confirm it wasn't my config, I removed my .config/nvim and .local/share/nvim, .local/state/nvim. I cloned a new lazyvim starter, and still experiencing the issue. 

If anyone else is experiencing this, disabling noice.nvim is a workaround:

```
return {
  { "folke/noice.nvim", enabled = false },
}
```

Edit: Downgrading to the non-head release also works:

```
return {
  { "folke/noice.nvim", tag = "v2.0.3" },
}
```
Currently trying to track which commit in noice is causing the problem; will report back here and an issue or PR on the noice repo if I find it.

**Comment by dpetka2001** [comment]
It's this [one](https://github.com/folke/noice.nvim/commit/4c2699111730a14144224d7b193bede6b707b1bc) which introduced the bug. The one before that works just fine.

**Reply by dusty-phillips** [reply, marked as answer]
Confirmed by independent simultaneous investigation. :-D 

Edit: Folke is working on this as we complain: https://github.com/folke/noice.nvim/commit/ee433a724c31858537a7d8c80d09c7686810da4a

**Reply by dpetka2001** [reply]
Yep, fixed already :smile:

**Reply by dusty-phillips** [reply]
And that commit is working. Thank you @folke! (and thank's to Guido for loaning folke his time machine?)

**Reply by folke** [reply]
Fixed indeed. Hopefully the last noice bug for a while now. I've been trying hard to keep up with all the upstream `vim.ui_attach` changes and quirks :)

**Comment by awilkins** [comment]
This seems to be back (verified with a fresh starter)

**Reply by dpetka2001** [reply]
Check the issues at `noice.nvim` github repo. There's one that's still open.
